ajax/libs/FiraCode/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "FiraCode",
+  "version": "1.204",
+  "filename": "fira_code.min.css",
+  "description": "Fira Code is an extension of the Fira Mono font containing a set of ligatures for common programming multi-character combinations.",
+  "homepage": "https://github.com/tonsky/FiraCode/wiki",
+  "keywords": [
+    "font",
+    "monospaces",
+    "programming",
+    "programming-ligatures",
+    "ligatures"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/tonsky/FiraCode"
+  },
+  "license": "OFL-1.1",
+  "author": "Nikita Prokopov <@tonsky> (http://tonsky.me/)",
+  "autoupdate": {
+    "source": "git",
+    "target": "https://github.com/tonsky/FiraCode.git",
+    "fileMap": [
+      {
+        "basePath": "distr",
+        "files": [
+          "**/*.!(html)"
+        ]
+      }
```

Comment:
Why not to add next pattern for old versions:
 {
         "basePath": "",
         "files": [
           "FiraCode-*"
         ]
       }
And add only 0.1 version manually
What do you think?

Reply:
@sashberd As maintainer said in [this PR](https://github.com/cdnjs/cdnjs/pull/11229#discussion_r145956817)
> if the non-dist basePath part is for the old version only, I won't suggest to use the flexible pattern, just use more static one, we don't want to have unexpected behavior in future.

So, I think It's not necessary to add root directory as the basePath just add old versions.
What do you think?

Thanks.

Reply:
OK